src/components/EventMap.js

```diff
@@ -226,7 +219,8 @@ export default function(store){
       },
 
       mapMounted() {
-        this.resetMap();
+        this.$refs.map.className = this.$refs.map.className.replace('-loading', '');
```

Comment:
(I would have liked for vue to be able to automatically manage a class which is in the initial HTML template but by poking around it appears that this does not work. I.e., because I want `.-loading` to be part of the initial HTML payload, it appears that vue, once it takes over that HTML, cannot automatically toggle that class's presence by binding it to some bit of data, for example `:class={ '-loading': this.isLoading }` )